addons/event/report/event_event_reports.xml

```diff
@@ -1,41 +1,82 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <record id="paperformat_euro_lowmargin" model="report.paperformat">
-        <field name="name">European A4 low margin</field>
-        <field name="default" eval="True"/>
+    <record id="paperformat_event_foldable_badge" model="report.paperformat">
+        <field name="name">Custom Paperformat for the Event Foldable Badge report</field>
+        <field name="default" eval="False"/>
+        <field name="disable_shrinking" eval="True"/>
         <field name="format">A4</field>
         <field name="page_height">0</field>
         <field name="page_width">0</field>
         <field name="orientation">Portrait</field>
-        <field name="margin_top">5</field>
-        <field name="margin_bottom">5</field>
-        <field name="margin_left">5</field>
-        <field name="margin_right">5</field>
-        <field name="header_line" eval="False"/>
-        <field name="header_spacing">0</field>
-        <field name="dpi">80</field>
+        <field name="margin_top">0</field>
+        <field name="margin_bottom">0</field>
+        <field name="margin_left">0</field>
+        <field name="margin_right">0</field>
+        <field name="dpi">96</field>
     </record>
 
-    <record id="report_event_registration_badge" model="ir.actions.report">
-        <field name="name">Registration Badge</field>
+    <record id="paperformat_event_full_page_ticket" model="report.paperformat">
+        <field name="name">Custom Paperformat for the Event Full Page Ticket report</field>
+        <field name="default" eval="False"/>
+        <field name="disable_shrinking" eval="True"/>
+        <field name="format">A4</field>
+        <field name="page_height">0</field>
+        <field name="page_width">0</field>
+        <field name="orientation">Portrait</field>
+        <field name="margin_top">0</field>
+        <field name="margin_bottom">8</field>
+        <field name="margin_left">0</field>
+        <field name="margin_right">0</field>
+        <field name="dpi">96</field>
+    </record>
+
+    <record id="action_report_event_registration_full_page_ticket" model="ir.actions.report">
```

Comment:
Note that apart that 'it contains more information', it is not really clear what are the use of those two reports ... Could be interesting to explain what they contain, what they do not contain and in which case they should be used.

Reply:
Cool !